src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,84 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 public class HelloWorldTest {
 
+    private RegistrationServiceImpl registrationService;
+    private User userLoginIsNull;
+    private User userLoginIsShorterThan6;
+    private User userPasswordIsNull;
+    private User userPasswordIsShorterThan6;
+    private User userAgeIsNull;
+    private User userAgeIsSmallestThan18;
+    private User checkExistingLogin1;
+    private User checkExistingLogin2;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        userLoginIsNull = new User(null, "1234567", 19);
+        userLoginIsShorterThan6 = new User("Oleks", "1234567", 19);
+        userPasswordIsNull = new User("Oleksandr", null, 19);
+        userPasswordIsShorterThan6 = new User("Oleksandr", "12345", 19);
+        userAgeIsNull = new User("Oleksandr", "1234567", null);
+        userAgeIsSmallestThan18 = new User("Oleksandr", "1234567", 17);
+        checkExistingLogin1 = new User("Oleksandr", "1234567", 18);
+        checkExistingLogin2 = new User("Oleksandr", "1234567", 18);
+    }
```

Comment:
Consider using `@BeforeAll` instead of `@BeforeEach` if the test data is not being modified in any test. This will create the test data once before all tests run, which is more efficient.